ci/build-docker.sh

```diff
@@ -71,9 +71,11 @@ cd "${BUILD_DIR}"
 ctest --stop-on-failure
 
 # Run the integration tests.
-(cd bigtable/tests && /v/bigtable/tests/run_integration_tests_emulator.sh)
-(cd bigtable/tests && /v/bigtable/examples/run_examples_emulator.sh)
-(cd storage/tests && /v/storage/tests/run_integration_tests.sh)
+for subdir in bigtable storage; do
+  echo
+  echo "${COLOR_RESET}Running integration tests for ${subdir}"
```

Comment:
Why do we need an explicit color reset here? Shouldn't every script reset the color themselves? Or are you assuming something else might end the script with color set in the terminal to something? And is that considered a bug?

Reply:
I was being paranoid.  Removed.